modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc

```diff
@@ -43,7 +43,7 @@ data:
       mirror-by-digest-only = true
 ----
 <1> The mirror registry’s certificate used when creating the mirror registry.
-<2> The configuration for the mirror registry.
+<2> The configuration file for the mirror registry. The mirror registry configuration adds `registries.conf` to `/etc/containers/registries.conf` in the Ignition config file of the Discovery image, and also to the `imageContentSources` section of the `install-config.yaml` file as a mirror. The Assisted Service pod converts `registries.conf` into an `imageContentSourcesPolicy` file, and uses it with the `--icsp` flag in a number of `oc adm` commands run against the Release image.
```

Comment:
This looks good, I would just recommend changing some wording. Maybe something like this (feel free to change it)...

The mirror registry configuration adds  mirror information to`/etc/containers/registries.conf` in the Discovery image. The mirror info is stored in the`imageContentSources` section of the `install-config.yaml` file when passed to the installer. The Assisted Service pod also converts `registries.conf` into an `ImageContentSourcePolicy` file, and uses it with the `--icsp-file` flag in a number of `oc` commands run against the Release image.